__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,16 @@
+const Itinerary = require('../src/Itinerary.js');
+const Port = require('../src/Port.js');
+
+describe('Itinerary', () => {
+    it('can be instatiated', () => {
+        expect(new Itinerary()).toBeInstanceOf(Object);
+    });
+    it('can have ports', () => {
```

Comment:
Its good practice to leave a clear line between each test to make the file easier to read
```
    });

    it('can have ports', () => {
```